src/components/Home.js

```diff
@@ -41,7 +42,9 @@ const Home = () => {
   return (
     <div>
       <h1>Welcome to Your Smart Shopping list!</h1>
-      <button onClick={createToken}>Create a new list</button>
+      <Button onClick={createToken} variant="contained" color="primary">
+        Create a new list
```

Comment:
This is an example of how to use one of the colors in the theme palette located in `/src/themes/theme.js`